webextensions/manifest/optional_permissions.json

```diff
@@ -255,7 +275,7 @@
             "description": "<code>debugger</code>",
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": false
```

Comment:
Chrome does support the debugger permission. Set this back to true.

Alternatively, if the objective is to drop permissions that are not on MDN from BCD, drop the `debugger` key.